lib/improver/cli/generate_topography_bands_mask.py

```diff
@@ -99,29 +99,55 @@ def main(argv=None):
         orography = load_cube(args.input_filepath_standard_orography)
         orography = next(orography.slices([orography.coord(axis='y'),
```

Comment:
Not sure if this bit of code should be in the function or not.

Reply:
I think this is to cover the case where there might be multiple realizations (with the same being true for the landmask below). I think the plugin itself should be able to figure out what to do if it has ensemble data, so I'd move it into the function.